src/phalanx/storage/kubernetes.py

```diff
@@ -0,0 +1,80 @@
+"""Storage layer for direct Kubernetes operations."""
+
+from __future__ import annotations
+
+from .command import Command
+
+__all__ = ["KubernetesStorage"]
+
+
+class KubernetesStorage:
```

Comment:
On the one hand, simpler...but on the other hand it feels like we already HAVE a lot of it mocked out for testing if we were using the Safir driver like we do in the Nublado controller.  I mean...we already have secrets and namespaces in there, right?

Reply:
Yes, the thing we don't have is waiting for a rollout to complete. I don't think that's a trivial thing to implement with timeout and error handling, etc., and we're doing everything else with command-line tools (Helm and Argo CD), so it didn't seem worth the effort to reimplement it in Python.

The testing code isn't that useful for Phalanx since Phalanx already tests the installer with a good integration test, making unit tests (particularly of this stuff) fairly pointless.